FF115 RTCRtpReceiver.jitterBufferTarget supported

hamishwillee opened the pull request:
FF115 added support for `RTCRtpReceiver.jitterBufferTarget` in https://bugzilla.mozilla.org/show_bug.cgi?id=1592988. This was caught by BCD collector and added to BCD in https://github.com/mdn/browser-compat-data/pull/22871

This adds docs for the property and an FF release note update.

hamishwillee committed:
FF115 RTCRtpReceiver.jitterBufferTarget supported

hamishwillee commented on files/en-us/web/api/rtcrtpreceiver/jitterbuffertarget/index.md:
```diff
@@ -0,0 +1,44 @@
+The actual user agent jitter buffer target will vary between maximum and minimum allowed values that reflects a target range that the user agent can provide based on network conditions and memory constraints, and can change at any time.
+The value returned by `jitterBufferTarget` is not affected by the actual target of the user agent.
+
+The change in average delay can be gradually observed over time by measuring the delta [`RTCInboundRtpStreamStats.jitterBufferDelay`](/en-US/docs/Web/API/RTCInboundRtpStreamStats) divided by the delta [`RTCInboundRtpStreamStats.jitterBufferEmittedCount`](/en-US/docs/Web/API/RTCInboundRtpStreamStats).
```
Note, these two stats are not documented yet, so I have chosen to link to the parent doc.

bsmth commented on files/en-us/web/api/rtcrtpreceiver/jitterbuffertarget/index.md:
```diff
@@ -0,0 +1,44 @@
+The actual user agent jitter buffer target will vary between maximum and minimum allowed values that reflects a target range that the user agent can provide based on network conditions and memory constraints, and can change at any time.
+The value returned by `jitterBufferTarget` is not affected by the actual target of the user agent.
+
+The change in average delay can be gradually observed over time by measuring the delta [`RTCInboundRtpStreamStats.jitterBufferDelay`](/en-US/docs/Web/API/RTCInboundRtpStreamStats) divided by the delta [`RTCInboundRtpStreamStats.jitterBufferEmittedCount`](/en-US/docs/Web/API/RTCInboundRtpStreamStats).
```
I was a bit confused with "delta" here (is it something an author should calculate?) and in fact in this section: https://www.w3.org/TR/webrtc-stats/#dom-rtcinboundrtpstreamstats-jitterbufferdelay

The spec says:

>The average jitter buffer delay can be calculated by dividing the [jitterBufferDelay](https://www.w3.org/TR/webrtc-stats/#dom-rtcinboundrtpstreamstats-jitterbufferdelay) with the [jitterBufferEmittedCount](https://www.w3.org/TR/webrtc-stats/#dom-rtcinboundrtpstreamstats-jitterbufferemittedcount).

Which seems simpler to me. What do you reckon?

hamishwillee replied:
So the spec tells you how to calculate the average buffer delay, but this is talking about how you can observe the _change_ in the average buffer delay - in order to see the effect of changing the target jitter buffer delay value. The fact that you found this confusing means it is poorly written. Adjusting.

```suggestion
The average jitter buffer delay can be calculated by dividing the [`RTCInboundRtpStreamStats.jitterBufferDelay`](/en-US/docs/Web/API/RTCInboundRtpStreamStats) by the [`RTCInboundRtpStreamStats.jitterBufferEmittedCount`](/en-US/docs/Web/API/RTCInboundRtpStreamStats).
In order to observe the effects of modifying the jitter buffer target you can track the value of this average over time.
```

bsmth reviewed (approved):
Looks great. One comment for you to look at regarding use of "delta" for calculating average delay (vs. change in average delay?) but leaving a +1 👍🏻

hamishwillee committed:
Update files/en-us/web/api/rtcrtpreceiver/jitterbuffertarget/index.md

hamishwillee commented:
Thanks for the review @bsmth - I modified the part you were concerned about (https://github.com/mdn/content/pull/33198#discussion_r1575868493) and I'm happy it addresses your comment, so merging.
But, if you disagree, ping me.

bsmth commented:
> Thanks for the review @bsmth - I modified the part you were concerned about ([#33198 (comment)](https://github.com/mdn/content/pull/33198#discussion_r1575868493)) and I'm happy it addresses your comment, so merging. But, if you disagree, ping me.

Looks good to me, let's merge 👍🏻
